tests/helper/helper_oc.go

```diff
@@ -669,6 +669,16 @@ func (oc OcRunner) GetAllPodsInNs(namespace string) string {
 	return Cmd(oc.path, ocArgs...).ShouldPass().Out()
 }
 
+//GetPodInNs gets the list of pods in given namespace. It waits for reasonable amount of time for pods to come up
+func (oc OcRunner) GetPodInNs(namespace string, regex string) string {
```

Comment:
It would help if the comment and function name gave some clue about this function trying to match a regex passed by the caller.

Reply:
Also, should we not have this in the CmdRunner interface and have kubectl implement this. We do want to get away from using specific runners right?

Looks good to me otherwise. Can lgtm once both these concerns are addressed

Reply:
@mohammedzee1000 @dharmit I have made the changes